aiconfig-docs/docs/gradio-workbook.md

```diff
@@ -0,0 +1,144 @@
+---
+sidebar_position: 21
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/tabConstants';
+
+# Gradio Workbooks
+
+**[Gradio Workbooks](https://huggingface.co/spaces/lastmileai/gradio-workbook-template) provide an out-of-the-box playground for creating Hugging Face Spaces.**
+
+The playground provides instant model selection, chaining of models, and templates - no setup or code required!
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/494523e1-23c6-4709-94cb-7ebe86873dfd" width="600"/>
+  </picture>
+</div>
+<br/>\
+Gradio Workbook unlocks new capabilities to HuggingFace Spaces users!
+
+1. **Build apps inspired by Spaces**. Gradio Workbook spaces are backed by an `aiconfig.json` file that stores prompts and model settings. Click **Download** to get the aiconfig file for your space. Use the `aiconfig.json` in your application code with the [AIConfig SDK](https://github.com/lastmile-ai/aiconfig).
+2. **Share read-only views of Spaces**. You can now share your workflow, prompts, and outputs, even as a space viewer. Check out this [space](https://huggingface.co/spaces/lastmileai/music_generator) and it's [read-only view](https://lastmileai.dev/aiconfig/cls2n2jpk00pnpe1n5xx8uub2)! Make edits to the space and click **Share Workbook** to get a read-only view of your own.
+
+## Quickstart
+
+### 1. Create a new space with Gradio Workbooks
+
+- **Start with a template**. Duplicate the [Gradio Workbook Quickstart space](https://huggingface.co/spaces/lastmileai/gradio-workbook-template)
+- **Create from scratch**. Create a [new Gradio SDK space](https://huggingface.co/new-space) and add these files to your space repo. To add files to your space repo, you can do so through the [Web UI](https://huggingface.co/docs/hub/repositories-getting-started#adding-files-to-a-repository-web-ui) or [terminal](https://huggingface.co/docs/hub/repositories-getting-started#terminal).
+  - [app.py](https://huggingface.co/spaces/lastmileai/gradio-workbook-template/blob/main/app.py)
+  - [requirements.txt](https://huggingface.co/spaces/lastmileai/gradio-workbook-template/blob/main/requirements.txt)
+
+### 2. Design your space
+
+- Add models and customize. Edits/changes won’t save automatically.
+
+### 3. Lock in your edits
+
+- Click **Download**, which downloads the `aiconfig` file. This captures the current configuration of your space.
+- Rename the downloaded file to `my_app.aiconfig.json`.
+- Replace `my_app.aiconfig.json` in the space repo with the downloaded one.
+
+Anyone who visits your space will see the state represented by `my_app.aiconfig.json`.
+
+## Examples
```

Comment:
General comments for these 3 examples:

1. Nit: Would prefer the URLs end with `:` instead of `.` because we're explaining what they do after
2. Make this resume-like, using action words. Ex: "Make your demo" can be more specific like "Implement various media formats (ex: text, audio, image) for multi-modal models" --> this is also somewhat a nit. I think the tone of "make your demo more captivating" isn't actually bad, and even more engaging for onboarding new people, just this playful tone should be consistent across the whole README doc if that's what we're going for